composer.json

```diff
@@ -27,7 +27,8 @@
 	},
 	"require-dev": {
 		"squizlabs/php_codesniffer": "~2.3",
-		"phpmd/phpmd": "~2.3"
+		"phpmd/phpmd": "~2.3",
+		"phpunit/phpunit": "~4.8"
```

Comment:
How does that make a difference? You'll just get the latest version anyway... it'd be different if it was in `require`, and it affected using packages, but it's not.